packages/core/core/src/RequestTracker.js

```diff
@@ -1516,18 +1542,36 @@ export default class RequestTracker {
         snapshotPath,
         opts,
       );
+
+      await storeRequestTrackerCacheInfo(this.options.cache, {
+        allLargeBlobKeys: Array.from(allLargeBlobKeys),
+        requestGraphKey,
+        snapshotKey,
+        timestamp: Date.now(),
+      });
+      report({
+        type: 'cache',
+        phase: 'end',
+        total,
+        size: this.graph.nodes.length,
+      });
     } catch (err) {
       // If we have aborted, ignore the error and continue
       if (!signal?.aborted) throw err;
-    }
 
-    await storeRequestTrackerCacheInfo(this.options.cache, {
-      allLargeBlobKeys: Array.from(allLargeBlobKeys),
-      requestGraphKey,
-      snapshotKey,
-      timestamp: Date.now(),
-    });
-    report({type: 'cache', phase: 'end', total, size: this.graph.nodes.length});
+      logger.warn({
+        origin: '@parcel/core',
+        message: 'Aborted writing to cache, ignoring error',
+        meta: {
+          errorMessage: err.message,
+          errorStack: err.stack,
+        },
+      });
+
+      for (let key of allLargeBlobKeys) {
+        await this.options.cache.deleteLargeBlob(key);
```

Comment:
Why do we need to delete all the large blobs on every bailout? The whole point of the bailout is to make responsive to user changes, this is potentially a lot of FS work we have to wait for?

Reply:
We don't need to but I'm pretty sure this can complete very fast (e.g.: thousands of ops a second & we only need to delete tens to one hundred file)

Reply:
also should we add this lint rule? https://eslint.org/docs/latest/rules/no-await-in-loop

Reply:
I'm not a fan of this rule because a lot of the time you're not looking for concurrency & you never want uncontrolled concurrency